corporate/views/portico.py

```diff
@@ -389,3 +391,57 @@ def remote_server_invoices_page(
 ) -> HttpResponseRedirect:
     list_invoices_session_url = billing_session.get_past_invoices_session_url()
     return HttpResponseRedirect(list_invoices_session_url)
+
+
+@zulip_login_required
+@typed_endpoint
+def customer_portal(
+    request: HttpRequest,
+    *,
+    return_to_billing_page: Json[bool] = False,
+    manual_license_management: Json[bool] = False,
+    tier: Optional[Json[int]] = None,
+) -> HttpResponseRedirect:
+    user = request.user
+    assert user.is_authenticated
+
+    if not user.has_billing_access:
+        return HttpResponseRedirect(reverse("billing_page"))
```

Comment:
I'm starting to think it might be worth adding a decorator `require_billing_admin` that does this step, since we have this pattern in a lot of billing views code.